kbase-extension/static/kbase/js/common/jobCommChannel.js

```diff
@@ -277,26 +294,6 @@ define([
             });
         }
 
-        convertJobState(backendJobState) {
```

Comment:
got rid of all this conversion; the FE now passes on what gets returned by the backend in nearly all situations